website/docs/guides/core-to-cloud-1.md

```diff
@@ -0,0 +1,245 @@
+  - If your dbt Core environment variables don’t follow this naming convention, perform a “find and replace” in your project to make sure all references to these environment variables contain the proper naming conventions.
+- dbt Cloud secures environment variables, offering additional measures for sensitive values, such as prefixing keys with `DBT_ENV_SECRET_` to obscure them in logs and the UI.
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/Environment Variables/project-environment-view.png" title="Setting project level and environment level values"/>
+
+### dbt Cloud environment variables order of precedence
+Environment variables in dbt Cloud are managed with a clear order of precedence, allowing users to define values at four levels (highest to lowest order of precedence):
+   - The job level (job override) or in the IDE for an individual developer (personal override). _(Highest precedence)_
+   - The environment level, which can be overridden by the job level or personal override.
+   - A project-wide default value, which can be overridden by the environment level, job level, or personal override.
+   - The optional default argument supplied to the `env_var` Jinja function in code. _(Lowest precedence)_
+  
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/Environment Variables/env-var-precdence.png" title="Environment variables order of precedence"/>
+
+### Set environment variables in dbt Cloud
+
+- To set these variables for an entire project or specific environments, navigate to **Deploy** > **Environments** > **Environment variables** tab.
+- To set these variables at the job level, navigate to **Deploy** > **Jobs** > **Select your job** > **Settings** > **Advanced settings**.
+- To set these variables at the personal override level, navigate to **Profile Settings** > **Credentials** > **Select your project** > **Environment variables**.
+
+## Orchestration setup
+
+This section outlines the considerations and methods to set up your dbt Cloud environments and jobs for orchestration. The following categories are covered in this section:
+
+- [dbt Cloud environments](/guides/core-to-cloud-1?step=8#dbt-cloud-environments-1)
+- [Initial setup steps](/guides/core-to-cloud-1?step=8#initial-set-up-steps-1)
+- [Additional configurations](/guides/core-to-cloud-1?step=8#additional-configurations)
+- [CI/CD setup](/guides/core-to-cloud-1?step=8#cicd-set-up)
+
+### dbt Cloud environments
+To use the [dbt Cloud's scheduler](/docs/deploy/job-scheduler), set up one environment as the production environment. This would be the [deployment](/docs/deploy/deploy-environments) environment. You can set up multiple environments for different stages of your deployment pipeline, such as development, staging/QA, and production.
+
+### Initial setup steps
+1. **dbt Core version** &mdash; In your environment settings, configure dbt Cloud with the same dbt Core version.
```

Comment:
Same note as further up that we may need to come back and update this step when all new Cloud accounts are versionless (and can't go back)